src/server/protocol.ts

```diff
@@ -216,6 +217,27 @@ namespace ts.server.protocol {
         projectFileName?: string;
     }
 
+    /**

```

Comment:
All of the *Request interfaces "request xxx information from the server" so I would just omit the comment.

Reply:
removed.